src/sentry/rules/processing/delayed_processing.py

```diff
@@ -502,8 +516,20 @@ def fire_rules(
             ).values()
 
             # TODO(cathy): add opposite of the FF organizations:workflow-engine-trigger-actions
+            sent = 0
             for callback, futures in callback_and_futures:
-                safe_execute(callback, groupevent, futures)
+                results = safe_execute(callback, groupevent, futures)
+                if results:
+                    sent += 1
```

Comment:
why is it saying this is unreachable 👀 `safe_execute` will return a result or none